.github/workflows/CI.yml

```diff
@@ -140,22 +148,24 @@ jobs:
       - uses: actions/checkout@v4
       - name: Setup Bun
         uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: 1.0.26
       - name: Setup node
         uses: actions/setup-node@v4
         with:
           node-version: 20
       - name: Install dependencies
         run: bun install
       - name: Download artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: bindings-${{ matrix.settings.target }}
           path: .
       - name: List packages
         run: ls -R .
         shell: bash
       - name: Test bindings
-        run: docker run --rm -v $(pwd):/build -w /build oven/bun:1 bun test
+        run: docker run --rm -v $(pwd):/build -w /build oven/bun:1.0.26 bun test
```

Comment:
i wish we could declare the bun version as a variable lol

side note: just looked this up, maybe we can do something like https://docs.github.com/en/actions/learn-github-actions/variables ?

Reply:
that will only work in this one line :'( https://github.com/orgs/community/discussions/25246

let's see if we can get it to work later!